ma/maShape.cc

```diff
@@ -814,6 +814,7 @@ void fixElementShapes(Adapt* a)
     return;
   double t0 = PCU_Time();
   int count = markBadQuality(a);
+  printf(" number of bad shape inside fixElementShape %d \n", count);
```

Comment:
@avinmoharana 

revert this file back to what it is on develop branch by using

`git checkout develop -- ma/maShape.cc`